### DISCUSSION TITLE
Question about Python in Unity builds

### DISCUSSION THREAD

**Discussion by flo-wolf** [exodrifter/unity-python#33] [Q&A] [answered]
Hello,
I am planning to execute pytorch code from within my unity build. In other words, I am trying to bundle python within my build, so that I only have to give the build to someone without the requirement of python to be installed on his machine. Would that be possible?

Also, pytorch requires Python 3.x, is that version supported?

Many thanks!

**Comment by exodrifter** [comment, marked as answer]
The general idea for including a Python library in a build is to place the Python library in the `StreamingAssets` folder, and then add the path to that library in the Python engine. However, that's a potential security problem (see #21).

Unfortunately, this library depends on IronPython which does not yet have support for Python 3.x. Only Python 2.x is supported at the moment. You can find the status of IronPython3 development [here](https://github.com/IronLanguages/ironpython3).

**Comment by exodrifter** [comment]
Hey there, it's been more than 30 days since I posted my response. I hope I've answered your question, but if you still have more related questions, please feel free to re-open this issue or open a new one.
